ethcore/src/pod_state.rs

```diff
@@ -89,7 +89,12 @@ mod test {
 
 	#[test]
 	fn create_delete() {
-		let a = PodState::from(map![ 1.into() => PodAccount::new(69.into(), 0.into(), vec![], map![]) ]);
+		let a = PodState::from(map![ 1.into() => PodAccount {
```

Comment:
Could be even simpler if there was `Default` for `PodAccount` (possibly with `#[cfg(test)]`)